pkg/kubelet/kubelet_pods.go

```diff
@@ -1114,11 +1114,11 @@ func (kl *Kubelet) HandlePodCleanups() error {
 	}
 	for _, runningPod := range runningRuntimePods {
 		switch workerState, ok := workingPods[runningPod.ID]; {
-		case ok && workerState == SyncPod, ok && workerState == TerminatingPod:
+		case ok && workerState == SyncPod, ok && workerState == TerminatingPod, ok && workerState == TerminatedPod:
 			// if the pod worker is already in charge of this pod, we don't need to do anything
 			continue
 		default:
-			// If the pod isn't in the set that should be running and isn't already terminating, terminate
+			// If the pod isn't in the set that should be running and isn't already terminating or terminated, terminate
```

Comment:
HandlePodCleanups func comment should update too. @tnqn

Reply:
I didn't find any outdated parts in HandlePodCleanups's comment. Could you be more specific which one should be updated?

Reply:
@tnqn including terminating --> including terminating or terminated
is better?

Reply:
https://github.com/kubernetes/kubernetes/blob/6669a016aec0f56b3e949858cd51503fb1b01a06/pkg/kubelet/kubelet_pods.go#L1037-L1039

I'm not grammar expert but I think the comment uses gerundial phrase to list "a series of cleanup work":
- terminat**ing** pod workers
- kill**ing** unwanted pods
- remov**ing** orphaned volumes/pod directories

"terminating" is one of the cleanup work.

Reply:
@tnqn That's right.